facet-jsonschema/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "facet-jsonschema"
+version.workspace = true
+authors.workspace = true
```

Comment:
Make sure to add yourself to authors. And probably don't even include me on that one because I haven't really done any work! Or I guess if we consider review work, just have me at the end of the authors list.